projects/go-lib/src/public_api.ts

```diff
@@ -44,6 +44,10 @@ export * from './lib/components/go-datepicker/calendar-cell.model';
 export * from './lib/components/go-datepicker/go-datepicker.component';
 export * from './lib/components/go-datepicker/go-datepicker.module';
 
+// File Upload
+export * from './lib/components/go-file-upload/go-file-upload.component';
+export * from './lib/components/go-file-upload/go-file-upload.module';
+
```

Comment:
If you decide to extract the DragonDrop out we'll want to make sure to export that as well.